modules/rosa-about-autoscaling-nodes.adoc

```diff
@@ -1,8 +1,8 @@
 // Module included in the following assemblies:
 //
-// rosa-nodes/rosa-enabling-autoscaling-nodes.adoc
+// rosa-nodes/rosa-enabling-cluster-autoscaling-nodes.adoc
 
-[id="rosa-enabling-autoscaling_{context}"]
+[id="rosa-enabling-cluster-autoscaling_{context}"]
```

Comment:
By editing this ID, you've now changed the file identifier. I looked through the repo and found the file in question. Your options are to keep the original ID name. Or you can rename the file....but you'll need to go rename the module and update the topic map with the correct new file name. The latter is the more labor intensive option.

Reply:
Ok - If I leave it as is, I think we have 2 files that are using the same ID.
I think that's why asciibinder is throwing a warning.
Is that ok? 

rosa-**about**-autoscaling-nodes.adoc has this header:
   // Module included in the following assemblies:
   //
   // **rosa-nodes/rosa-enabling-autoscaling-nodes.adoc**

   [id="rosa-enabling-autoscaling_{context}"]
   = About autoscaling nodes in a cluster

rosa-**enabling**-autoscaling-nodes has this header:
   // Module included in the following assemblies:
   //
   // **rosa-nodes/rosa-enabling-autoscaling-nodes.adoc**

   [id="rosa-enabling-autoscaling_{context}"]
   = Enabling autoscaling nodes in an existing cluster

Reply:
@sfortner-RH  Maybe those duplications are as they should be. (?)

Reply:
Agh! I see the issue! This is your overall "About" module. So the ID needs to be "rosa-about-autoscaling-nodes" to match the filename. This will resolve the conflict. 

The module named "rosa-enabling-autoscaling' is a task module that follows this one and should remain untouched.

Does that make sense?

Reply:
Ah, yes - I see it now. Thanks!

Reply:
If the id is supposed to match the filename, should I change the id= in the rosa-enabling-autoscaling-nodes file too? It looks like it is missing "-nodes"

**rosa-enabling-autoscaling-nodes** has this header:
// Module included in the following assemblies:
//
// rosa-nodes/rosa-enabling-autoscaling-nodes.adoc

[id="**rosa-enabling-autoscaling**_{context}"]
= Enabling autoscaling nodes in an existing cluster